curriculum/challenges/english/07-scientific-computing-with-python/learn-tree-traversal-by-building-a-binary-search-tree/65c64057a080baa7d60523ed.md

```diff
@@ -1,8 +1,8 @@
 ---
-id: 655a16db1ccc5e132b5bc44d
-title: Step 21
+id: 65c64057a080baa7d60523ed
+title: Step 25
```

Comment:
can this step be merged into step 24? or step 24 becomes too complex then?

Reply:
Step-24 would be too wordy if we merge it with 25.